### PR TITLE
Add schedule card and extract Card Details component for reuse between Content card and Schedule card

### PR DESCRIPTION
**Pull request by igilfillan**
As of this PR we now will show the channel icon if a channel is passed to either card, before this would only show if the card type was live. This will be a problem if you are passing the channel into the content card when you don't want the icon shown. So now it's the consumers responsibility to not pass that if it's not wanted.

### CODE REVIEW

File: src/components/Card/ScheduleCard.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  display: block;
+  position: absolute;
+  opacity: 0.35;
+
+  :before {
+    content: '';
+    position: absolute;
+    z-index: 1;
+    height: 100%;
+    width: 101%;
```
Comment (davidgarry) [inline src/components/Card/ScheduleCard.js]: Why 101% ?

File: src/components/Card/ScheduleCard.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+const ScheduleCard = ({ card, ...props }) => (
+  <StyledWrapper {...props}>
+    <StyledImage img={card.img} />
+    <StyledCardDetails card={card} />
+  </StyledWrapper>
+);
+
+ScheduleCard.propTypes = {
+  card: PropTypes.shape({
```
Comment (davidgarry) [inline src/components/Card/ScheduleCard.js]: The `card` prop should be required no ?

File: src/components/Card/ScheduleCard.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    content: '';
+    position: absolute;
+    z-index: 1;
+    height: 100%;
+    width: 101%;
+    background: linear-gradient(
+      180deg,
+      rgba(21, 23, 38, 0) 0%,
+      rgba(21, 23, 38, 0.3) 18.66%,
+      rgba(21, 23, 38, 0.85) 100%
```
Comment (davidgarry) [inline src/components/Card/ScheduleCard.js]: Can you export this color in colors.js ?

File: src/components/Card/ContentCard.js [hunk 4]
```diff
@@ -126,17 +72,17 @@ export const StyledLiveLabel = styled.div`
...
 
-const StyledTimeStamp = styled.div`
-  padding: 8px 0;
-  margin: 0;
-  color: ${colors.manatee};
-  font-size: 12px;
-`;
-
 const ContentCard = ({ card, type, ...props }) => {
-  const { img, url, category, title, description, timestamp, channel, liveLabel } = card;
```
Comment (davidgarry) [inline src/components/Card/ContentCard.js]: Maybe you can use this syntax : 
`const { img, url, liveLabel, ...cardDetailsProps } = card;`

And remove the definition of `cardDetailsProps` below.

File: src/components/Card/CardDetails.js [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+          {channel && <ChannelIcon height={15} type={channel} />}
+          <StyledTimeStamp>{timestamp}</StyledTimeStamp>
+        </StyledDetails>
+      </StyledFooter>
+    </StyledContent>
+  );
+};
+
+CardDetails.propTypes = {
+  card: PropTypes.shape({
```
Comment (davidgarry) [inline src/components/Card/CardDetails.js]: `card` should be required

File: src/components/Card/ScheduleCard.js (hunk 1 above)
Comment (ldabiralai) [reply src/components/Card/ScheduleCard.js]: @JoffreyMaheo had the same comment a while ago. Without that the gradient doesn't seem to fill the whole width of the card.

image

File: src/components/Card/ScheduleCard.js (hunk 2 above)
Comment (ldabiralai) [reply src/components/Card/ScheduleCard.js]: Good point, done!

File: src/components/Card/CardDetails.js (hunk 5 above)
Comment (ldabiralai) [reply src/components/Card/CardDetails.js]: Done!

File: src/components/Card/ScheduleCard.js (hunk 3 above)
Comment (ldabiralai) [reply src/components/Card/ScheduleCard.js]: Done!

**Pull request merged**